tweet_trigger_readme.md

```diff
@@ -0,0 +1,26 @@
+tweet_trigger_readme.md
+
+Introduction
+------------
+
+Tweet_trigger is an application which collects earthquake events that come in through PDL and tweets them from the "notedalerts" account. It was designed to run in Python 3. 
```

Comment:
regarding the twitter account, insteawd of explicitly listing "notedalerts" I would say something like "and tweets them from a configurable twitter account. For testing purposes, the notedalerts twitter account is used. It was designed to...."